knox/test_knox.py

```diff
@@ -29,9 +29,9 @@ def _run_test_script(self, name, cert_type):
                               ("STANDARD", ["m", "w-0"]),
                               ("HA", ["m-2", "w-0"]))
     def test_knox_localhost_cert(self, configuration, machine_suffixes):
-        # Init action supported on Dataproc 1.3+
-        if self.getImageVersion() < pkg_resources.parse_version("1.3"):
-            self.skipTest("Not supported in pre 1.3 images")
+        if self.getImageOs() == 'debian':
+            if self.getImageVersion() <= pkg_resources.parse_version("1.4"):
```

Comment:
maybe chage this to 
self.getImageVersion() < pkg_resources.parse_version("1.5")

Reply:
I wanted to use `<=` so it will be simpler to clean up code base when we deprecate 1.4 images